src/features/wallet/actions/mintFarm.ts

```diff
@@ -31,3 +34,43 @@ export async function mintFarm(request: Request) {
 
   return {};
 }
+
+export async function mintNFTFarm(request: Request) {
+  const response = await window.fetch(`${API_URL}/mint-account/${request.id}`, {
+    method: "POST",
+    headers: {
+      "content-type": "application/json;charset=UTF-8",
+      "X-Transaction-ID": request.transactionId,
+      Authorization: `Bearer ${request.jwt}`,
+    },
+  });
+
+  if (response.status >= 400) {
+    throw new Error(ERRORS.LOGIN_SERVER_ERROR);
+  }
```

Comment:
```suggestion
  if (response.status >= 400) {
    throw new Error(ERRORS.MINT_SERVER_ERROR);
  }
```

The team can get discombobbed when these are wrong